src/layouts/BaseLayout.astro

```diff
@@ -15,6 +15,16 @@ const textColor = config.branding.contrast_color;
 const backgroundColor = config.branding.background_color;
 
 const favicon = `/${config.branding.favicon}` || '/favicon.svg';
+
+function truncateString(str: string, maxLength: number) {
+  if (str.length > maxLength) {
+    return str.substring(0, maxLength) + '...';
```

Comment:
I wonder if that's necessary? Browser tab titles will truncated anyway. We may just want to ensure we're not hitting some spec-defined limit. (Not sure if one exists, but we could e.g. cut at 256 or 1024 chars?)

Reply:
Based on this:

https://www.w3.org/Provider/Style/TITLE.html

I will limit the length to 64 (actually 61 plus the '...')